### PR TITLE
Phase 2!

### PR DESCRIPTION
**Pull request by sgoodwin**
This sets up the new tab order where the middle tab is now Coverage.
Havard is working on the much nicer looking UI that will be used here,
but this allows you to see the general idea. You can now go through
onboarding for singapore and norway separately. This selection of
countries is fairly arbitrary to show that onboarding can be summoned
for any region we expect the server to support.

### CODE REVIEW

File: dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift [hunk 1]
```diff
@@ -25,7 +25,7 @@ class NorwayUserHappyFlowWithJumioStageDeciderTests: XCTestCase {
     
     func testUserHasCompletedJumio() {
         let decider = StageDecider()
-        let localContext = LocalContext(hasSeenNotificationPermissions: true, hasSeenVerifyIdentifyOnboarding: true, hasCompletedJumio: true, hasSeenLocationPermissions: true)
+        let localContext = LocalContext(hasSeenNotificationPermissions: true, hasCompletedJumio: true, hasSeenLocationPermissions: true)
```
Comment (la3lma) [inline dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift]: Much repeated code her.  Is it possible to delegate this either to a method or to the fixture setup for the test suite?

File: ostelco-core/Models/StageDecider.swift [hunk 2]
```diff
@@ -38,11 +37,10 @@ public class LocalContext {
...
+    public init(hasFirebaseToken: Bool = false, hasAgreedToTerms: Bool = false, hasSeenNotificationPermissions: Bool = false, selectedVerificationOption: IdentityVerificationOption? = nil, myInfoCode: String? = nil, hasSeenESimOnboarding: Bool = false, hasSeenESIMInstructions: Bool = false, hasSeenAwesome: Bool = false, hasCompletedJumio: Bool = false, hasCompletedAddress: Bool = false, serverIsUnreachable: Bool = false, locationProblem: LocationProblem? = nil, hasSeenRegionOnboarding: Bool = false, hasSeenLocationPermissions: Bool = false, hasCameraProblem: Bool = false) {
```
Comment (la3lma) [inline ostelco-core/Models/StageDecider.swift]: Very long line. Hard to read.  I prefer to have each parameter on a line, and any parameters (if any) that are worthy of special attention pointed out in comments.

File: ostelco-ios-client/ViewControllers/Country/CoverageViewController.swift [hunk 3]
```diff
@@ -0,0 +1,72 @@
+//
+//  CoverageViewController.swift
+//  ostelco-ios-client
+//
+//  Created by Samuel Goodwin on 10/1/19.
+//  Copyright © 2019 mac. All rights reserved.
```
Comment (la3lma) [inline ostelco-ios-client/ViewControllers/Country/CoverageViewController.swift]: This isn't strictly true :-). The file headers should reflect that this is red otter software.

Comment (havard024) [reply ostelco-ios-client/ViewControllers/Country/CoverageViewController.swift]: our IDE (xcode) creates this, should be configurable though (I assume). I'm creating a task in notion to remember this.

File: dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift (hunk 1 above)
Comment (havard024) [reply dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift]: Created a task in notion to address this

File: ostelco-core/Models/StageDecider.swift (hunk 2 above)
Comment (sgoodwin) [reply ostelco-core/Models/StageDecider.swift]: Yes we've discussed this already. This object is about to be simplified so it won't be quite so absurd.

File: dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift (hunk 1 above)
Comment (sgoodwin) [reply dev-ostelco-ios-clientTests/CoordinatorTests/NorwayUserHappyFlowWithJumioStageDeciderTests.swift]: This isn't repeated code, it's just the data the test uses. There's no logic being duplicated here. Fixtures would only make this even more complicated.

File: ostelco-core/Models/StageDecider.swift (hunk 2 above)
Comment (la3lma) [reply ostelco-core/Models/StageDecider.swift]: (also I have very little object permanence when it comes to this codebase.  I mostly recognise only what I see in front of me at the moment, if it connects to or even if something I've seen a few days ago I will just not notice :-).

Comment (sgoodwin) [reply ostelco-core/Models/StageDecider.swift]: Ok, but if you keep making the same comments and I keep having to respond to explain why it's not an issue, that's a bit frustrating for me. So maybe you could stop?

**Pull request merged**